fem/moonolith/mortarintegrator.hpp

```diff
@@ -114,9 +116,56 @@ class VectorL2MortarIntegrator : public MortarIntegrator
                               ElementTransformation &test_Trans,
                               DenseMatrix &elemmat) override;
 
-   inline bool is_vector_fe() const override { return true; }
+private:
+   Coefficient *Q;
+   VectorCoefficient *VQ;
+   MatrixCoefficient *MQ;
+
+   void Init(Coefficient *q, VectorCoefficient *vq, MatrixCoefficient *mq)
+   {
+      Q = q;
+      VQ = vq;
+      MQ = mq;
+   }
+};
+
+/*!
+ * @brief Integrator for Lagrange vector finite elements. Experimental.
+ * $$ (u, v)_{L^2(\mathcal{T}_m \cap \mathcal{T}_s)}, u \in U(\mathcal{T}_m )
+ * and v \in V(\mathcal{T}_s ) $$
+ */
+class TPL2MortarIntegrator : public MortarIntegrator
```

Comment:
What does TP stand for?

Reply:
Changed the name of "TPL2MortarIntegrator" to "LagrangeVectorL2MortarIntegrator"